docs/pages/contributing/documentation.mdx

```diff
@@ -24,16 +24,14 @@ the kind of documentation we want to transform our current documentation into.
 
 </Admonition>
 
-- [How to Contribute to Teleport's
-  Documentation](./documentation/how-to-contribute.mdx): Follow this guide to
-  start contributing changes to Teleport's documentation.
-- [How to Review Documentation Changes](./documentation/reviewing-docs.mdx):
-  Read our recommended approach to preview docs changes in order to review pull
+- [How to contribute to Teleport documentation](./documentation/how-to-contribute.mdx) describes 
+  how to set up a local environment and contribute changes to Teleport documentation.
+- [How to review documentation changes](./documentation/reviewing-docs.mdx) explains how to
```

Comment:
Since we use title case for titles, should the text of the links in this list use title case as well?

Reply:
For the sake of readability, sentence case is better in almost any context. But if the current convention uses Headline Style, I can be flexible.